src/meta/buying_options.yaml

```diff
@@ -1,18 +1,82 @@
-carrot:
+golden_carrot:
   type: item
   name: Golden Carrots
   item: minecraft:golden_carrot
   price: 200
   amount: 8
+torch:
+  type: item
+  name: Torches
+  item: minecraft:torch
+  price: 100
+  amount: 32
+hopper:
+  type: item
+  name: Hoppers
+  item: minecraft:hopper
+  price: 500
+  amount: 16
+cobblestone:
+  type: item
+  name: Cobblestone
+  item: minecraft:cobblestone
+  price: 10
```

Comment:
```suggestion
  price: 100
```

Reply:
that seems like a lot just for a stack of blocks

Reply:
Consider that a single diamond is worth ~7k atm. That's 700 stacks of cobble stone. I think a diamond is worth like 70 stacks of cobblestone, rather than 700.

Cobble also gives 1 GRK/item in the shredder, so that would give people an infinite loop of profit (neglecting the limits).

Costing less than 60 doesn't make sense, and I think there's a convenience factor at play, which makes 100 GRK a pretty good deal. I'd use it over mining the blocks myself.